packages/react-components/react-theme/stories/Theme/motion/index.stories.mdx

```diff
@@ -0,0 +1,18 @@
+import { Meta, Canvas } from '@storybook/addon-docs';
```

Comment:
What is the reason to have the stories in multiple folders with indexes everywhere? Just to achieve the intended menu structure?

I like the proposed menu structure, just a note - that's what we used to have and then @micahgodbolt reimplemented it - please confirm with him so that we do not run round in circles.

Reply:
not sure why I consolidated. might have been the move to index files and not wanting to set up an index file for every single one of those topics. I'm fine with this change. It exposes the topics more clearly.

Reply:
I was following the tsx component stories approach.  Micah explained that the folders are strictly necessary.  We agree that it does help organize as the mdx depends on tsx.  We can always flatten later if we find it is an annoying structure.